cwms-data-api/src/main/java/cwms/cda/data/dao/LocationsDao.java

```diff
@@ -35,6 +36,8 @@ public interface LocationsDao {
 
     List<Location> getLocations(String names, String units, String datum, String officeId);
 
+    List<CwmsIdLocationKind> getLocationKinds(String names, String units, String datum, String officeId);
```

Comment:
Can you rename "names" to something else.  The old-school pl/sql used to have a field called names that was a pipe delimitied list of ids.  I think some other parameter name would be better.  Not sure what, regex, idRegex, idPattern, mask - I'm sure you can come up with something slightly cleaner